error_type: ValidationError
error_module: pydantic_core._pydantic_core
message: "3 validation errors for NodeManagerInputState\noptional_field\n  Input should\
  \ be a valid dictionary or instance of OnexFieldModel [type=model_type, input_value=None,\
  \ input_type=NoneType]\n    For further information visit https://errors.pydantic.dev/2.11/v/model_type\n\
  event_id\n  Value error, event_id must be a valid UUID string [type=value_error,\
  \ input_value='test-event-0013', input_type=str]\n    For further information visit\
  \ https://errors.pydantic.dev/2.11/v/value_error\nnode_version\n  Input should be\
  \ a valid dictionary or instance of SemVerModel [type=model_type, input_value='1.0.0',\
  \ input_type=str]\n    For further information visit https://errors.pydantic.dev/2.11/v/model_type"
validation_errors:
- type: model_type
  loc:
  - optional_field
  msg: Input should be a valid dictionary or instance of OnexFieldModel
  input: null
  ctx:
    class_name: OnexFieldModel
  url: https://errors.pydantic.dev/2.11/v/model_type
- type: value_error
  loc:
  - event_id
  msg: Value error, event_id must be a valid UUID string
  input: test-event-0013
  ctx:
    error:
      error_type: ValueError
      error_module: builtins
      message: event_id must be a valid UUID string
  url: https://errors.pydantic.dev/2.11/v/value_error
- type: model_type
  loc:
  - node_version
  msg: Input should be a valid dictionary or instance of SemVerModel
  input: 1.0.0
  ctx:
    class_name: SemVerModel
  url: https://errors.pydantic.dev/2.11/v/model_type
